build_image.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ $# -eq 0 ]
+  then
+    echo "No argument supplied. Define either 'virtual' or 'enclave'";
+    exit
+fi
+
+if [ "$1" != "virtual" -a "$1" != "enclave" ];then 
+    echo "Wrong argument value. Define either 'virtual' or 'enclave'";
+    exit
+fi
+
+setup_type=$1
+app_path=$PWD
+app_name=${app_path##*/}
+parent_path=`dirname $PWD`
+
+echo "-- Generating image for $setup_type" container.
+
+mkdir -p $app_path/app_certificates
```

Comment:
I wonder whether this would be better in a folder called "workspace/docker_certificates" or something similar and then it is similar to the way it works with the sandbox

Reply:
Nice suggestion! :)
done.